mne/channels/channels.py

```diff
@@ -1762,3 +1766,180 @@ def combine_channels(inst, groups, method='mean', keep_stim=False,
+    # temporal lobes. Here we switch the sides for these outliers. For other
+    # lobes it is not a big problem because of the vicinity of the lobes.
+    with np.errstate(invalid='ignore'):  # invalid division, greater compare
+        zs = np.abs(zscore(x[rt]))
+        outliers = np.array(rt)[np.where(zs > 2.)[0]]
+    rt = list(np.setdiff1d(rt, outliers))
+
+    with np.errstate(invalid='ignore'):  # invalid division, greater compare
+        zs = np.abs(zscore(x[lt]))
+        outliers = np.append(outliers, (np.array(lt)[np.where(zs > 2.)[0]]))
+    lt = list(np.setdiff1d(lt, outliers))
+
+    l_mean = np.mean(x[lt])
+    r_mean = np.mean(x[rt])
+    for outlier in outliers:
+        if abs(l_mean - x[outlier]) < abs(r_mean - x[outlier]):
+            lt.append(outlier)
+        else:
+            rt.append(outlier)
+
+    if add_stim:
+        stim_ch = _get_stim_channel(None, info, raise_error=False)
+        if len(stim_ch) > 0:
+            for region in [lf, rf, lo, ro, lp, rp, lt, rt]:
+                region.append(info['ch_names'].index(stim_ch[0]))
+    return OrderedDict([('Left-frontal', lf), ('Right-frontal', rf),
+                        ('Left-parietal', lp), ('Right-parietal', rp),
+                        ('Left-occipital', lo), ('Right-occipital', ro),
+                        ('Left-temporal', lt), ('Right-temporal', rt)])
+
+
+def _divide_side(lobe, x):
+    """Make a separation between left and right lobe evenly."""
+    lobe = np.asarray(lobe)
+    median = np.median(x[lobe])
+
+    left = lobe[np.where(x[lobe] < median)[0]]
+    right = lobe[np.where(x[lobe] > median)[0]]
+    medians = np.where(x[lobe] == median)[0]
+
+    left = np.sort(np.concatenate([left, lobe[medians[1::2]]]))
+    right = np.sort(np.concatenate([right, lobe[medians[::2]]]))
+    return list(left), list(right)
+
+
+@verbose
+def read_vectorview_selection(name, fname=None, info=None, verbose=None):
+    """Read Neuromag Vector View channel selection from a file.
+
+    By default, the selections used in ``mne_browse_raw`` are supported.
+    Additional selections can be added by specifying a selection file (e.g.
+    produced using ``mne_browse_raw``) using the ``fname`` parameter.
+
+    The ``name`` parameter can be a string or a list of string. The returned
+    selection will be the combination of all selections in the file where
+    (at least) one element in name is a substring of the selection name in
+    the file. For example, ``name=['temporal', 'Right-frontal']`` will produce
+    a combination of ``'Left-temporal'``, ``'Right-temporal'``, and
+    ``'Right-frontal'``.
+
+    The included selections are:
+
+        * ``'Vertex'``
+        * ``'Left-temporal'``
+        * ``'Right-temporal'``
+        * ``'Left-parietal'``
+        * ``'Right-parietal'``
+        * ``'Left-occipital'``
+        * ``'Right-occipital'``
+        * ``'Left-frontal'``
+        * ``'Right-frontal'``
```

Comment:
All of this seems more appropriate for a `Notes` section (I know you copy-pasted, but since all lines are new/changed it's not a bad time to do it anyway)

Reply:
note also that I think these bullet points are indented too far, they will end up rendered as if they were bullets within a blockquote (as often happens in our docs)

Reply:
> All of this seems more appropriate for a `Notes` section

Personally, I think this is a bit of an anti-pattern: without knowing the names of the selections, users cannot use the function. Forcing them to scroll down to the Notes section doesn't seem very user-friendly, so I'd rather leave it here.



> note also that I think these bullet points are indented too far

Thanks, I will have a look.

Reply:
> Personally, I think this is a bit of an anti-pattern: without knowing the names of the selections, users cannot use the function. Forcing them to scroll down to the Notes section doesn't seem very user-friendly, so I'd rather leave it here.

I still think these should be moved out of the top, for the following reasons:

- The top should be reserved for a hopefully short explanation of the function. That way the parameter list at the top is close to the definition list below it and people don't have to scroll a ton (or back and forth) to see them.
- The argument that some critical parameters should have their options described at the top does not generalize well. Imagine applying this logic to something like `maxwell_filter` where you arguably need to know about what options are available for half a dozen arguments and what those options mean to effectively use it.

To me `Notes` is a reasonable place for it, but another option would be to move them to a non-bulleted list in the description of the parameter `name`. This fits with the general pattern that parameter options should be described (compactly) within the parameter definition list itself, or in Notes for details / if there are too many options.

Reply:
> To me `Notes` is a reasonable place for it, but another option would be to move them to a non-bulleted list in the description of the parameter `name`. This fits with the general pattern that parameter options should be described (compactly) within the parameter definition list itself, or in Notes for details / if there are too many options.

I like the idea of moving it to the parameter description. To me, Notes is more like implementation details a normal user shouldn't have to worry about

Reply:
I've addressed this in f470d86